### PR TITLE
Assoc

### PR DESCRIPTION
**Pull request by enis-memic**
### For PR I implemented following:

- Created Classroom class

- Create the has-many/belongs-to relationship between Classroom and Student.

- Created a class Book.

- Created a class Rental.

- Created the many-to-many between Person, Book and Rental.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def rent_book(person, date)
+    Rental.new(date, person, self)
```
Comment (TedLivist) [inline person.rb]: - [x] We already have access to the `person` object through `self`, so what we need is `book`, not `person`.

- [x] Kindly take note of the arrangement for the parameters.

  ```
  rent_book(book, date)
    Rental.new(date, book, self)
  ```

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_book(person, date)
+    Rental.new(date, person, self)
```
Comment (TedLivist) [inline book.rb]: - [x] We already have access to the `book` object through `self`, so what we need is `person`, not `book`.

- [x] Kindly take note of the arrangement for the parameters.

  ```
  def rent_book(person, date)
    Rental.new(date, self, person)
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @enis-memic,

Good job so far!
✔️ Establishment of associations
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (enis-memic) [reply person.rb]: Thank you for the tip.

File: book.rb (hunk 2 above)
Comment (enis-memic) [reply book.rb]: Thank you for the tip.

**Review by amejid — APPROVED** [review]
Hi @enis-memic, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter check passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Associations established between classes :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**